plugin-api/src/main/java/org/hyperledger/besu/plugin/services/metrics/IbftQueries.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.plugin.services.metrics;
+
+import org.hyperledger.besu.plugin.data.Address;
+
+import java.util.Collection;
+
+public interface IbftQueries extends PoAMetricsService {
+
+  public int getRoundNumberFromCanonicalHead();
```

Comment:
I like this as a convenience method but I think it's a bit of an anti-pattern we're developing with only being able to get information about the head block in these PoA APIs.  We really should have a variant that accepts a `BlockHeader` so you can ask for the round number for any block, not just the canonical head.  Also applies to `PoAMetricsService.getValidatorsForLatestBlock`.  And the naming between the two should be consistent as well - CanonicalHead is probably the right name so maybe need to make a note to rename `getValidatorsForLatestBlock` to `getValidatorsFromCanonicalHead` at the same time as we rename `PoAMetricsService` to remove "metrics" from the name.